jest.config.js

```diff
@@ -1,11 +1,8 @@
 module.exports = {
-  moduleFileExtensions: ['js'],
+  moduleFileExtensions: ["js"],
```

Comment:
Not a fan of doublequotes for JS strings personally and our previous projects have been with `'`. More important to be consistent though in either case.